Sources/JXKit/JXCoding.swift

```diff
@@ -14,26 +14,6 @@ extension JXValue {
     }
 }
 
-extension JXValue {
```

Comment:
Moved these to public funds in JXValue as part of its array support. Appended "Element" to the names to be consistent with existing setElement.